### ISSUE TITLE
common_name in go-reference-species.yaml is not unique?

### ISSUE THREAD

**Issue by kltm** [geneontology/go-site#1564]
common_name in go-reference-species.yaml is not unique. Is this the intention.

For example, "bacteria" is list 18 times: https://github.com/geneontology/go-site/blob/master/metadata/go-reference-species.yaml

Related to: https://github.com/geneontology/go-site/issues/1522

Either way, the fix is here: https://github.com/geneontology/go-site/blob/d415f978b75a12b3e80a6e9d43c72eba4405c03b/metadata/go-reference-species.schema.yaml#L17

Tagging @dustine32

**Comment by dustine32** [comment]
@kltm This "not-unique" fix works for us after talking with @huaiyumi and @thomaspd. Some of the `common_name` values will eventually need to be manually curated/updated in PANTHER at a later point but this field is not required to be unique.

**Comment by kltm** [comment]
Fair enough. I think it would be good to try and get them a bit more specific, but not a blocker for now.

**Comment by pgaudet** [comment]
@kltm - did you want to use this for anything ? or was this just out of curiosity ? 

If needed we can improve this.

**Comment by pgaudet** [comment]
@mugitty shared the source of the names: http://pantherdb.org/panther/summaryStats.jsp

**Comment by pgaudet** [comment]
We'll try to improve this

**Comment by dustine32** [comment]
@pgaudet @huaiyumi @thomaspd Here's my quick attempt at improving this. You can easily compare the changes by looking at this https://github.com/geneontology/go-site/commit/8fbdffabcf2817e1a5e88bd1748037960ae261db commit.

All I did was alter which PANTHER DB field is pulled to match the same field that populates the "Common Name" column in http://pantherdb.org/panther/summaryStats.jsp.

If any further changes to these values are needed they should probably be fixed in the PANTHER DB tables. To do that, I can dump out the organism table for 15.0, someone can manually change the name, then I'll update back into the DB.

**Comment by kltm** [comment]
@dustine32 Does this mean that they are unique (and that we can update the schema)?

**Comment by kltm** [comment]
@pgaudet This will be used for the source-of-truth for the species we support, so we want to have the label, etc., standardized as well.

**Comment by dustine32** [comment]
@kltm I wish, "aspergillus" appears twice. Once in Emericella nidulans and again in Neosartorya fumigata.

I'm gonna push that "should-be-unique" question to @pgaudet @huaiyumi @thomaspd unless there's an immediate need for this column to be unique.

**Comment by pgaudet** [comment]
Some suggestions to make names unique: 

* Emericella nidulans  -> a nidulans (note that the official name is Aspergillus nidulans, see https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=162425) 
* Neosartorya fumigata -> a fumigatus (official name Aspergillus fumigatus, see https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=746128)

plus
* Arabidopsis thaliana -> mouse-ear cress

Thanks, Pascale
